packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -247,9 +248,33 @@ export class MessageWidget extends Component {
       ) {
         // Send notification of new message
         const avatars = props.avatar.get('items');
+        let cardsLength = 0;
+        let message = lastActivityFromThis.object.displayName;
+
+        if (Object.prototype.hasOwnProperty.call(lastActivityFromThis, 'object')
+        && Object.prototype.hasOwnProperty.call(lastActivityFromThis.object, 'cards') && lastActivityFromThis.object.cards.length > 0) {
+          cardsLength = lastActivityFromThis.object.cards.length;
+          const sharedCardsNotificationFormat = formatMessage(messages.sharedCards, {cardsLength});
+
+          if (lastActivityFromThis.verb === 'post') {
+            message = sharedCardsNotificationFormat;
+          }
+          else if (lastActivityFromThis.verb === 'share' && Object.prototype.hasOwnProperty.call(lastActivityFromThis.object, 'files') && Object.prototype.hasOwnProperty.call(lastActivityFromThis.object.files, 'items')
+          && lastActivityFromThis.object.contentCategory === 'images') {
+            const cardImagesLength = lastActivityFromThis.object.files.items.filter((item) => item.type === 'cardReference').length;
+            const itemsLength = lastActivityFromThis.object.files.items.length;
+            const sharedPhotosNotificationFormat
```

Comment:
It might make it more readable for this variable to be named `sharedPhotosNotificationMessage` instead.